tutormfe/plugin.py

```diff
@@ -93,14 +94,25 @@ def iter_mfes() -> t.Iterable[tuple[str, MFE_ATTRS_TYPE]]:
     """
     yield from MFE_APPS.apply({}).items()
 
+def iter_domains(additional_domains, main_host, main_theme):
+    """
+    Yield:
+
+        (name, theme)
+    """
+    yield main_host, main_theme
+    for domain_config in additional_domains:
+        if 'mfe_proxy' in domain_config and 'mfe_theme' in domain_config:
```

Comment:
@xitij2000 Without a comment these two conditions seem arbitrary. Could you add a comment explaining why we are checking for these keys in the configurations?